src/Infrastructure/Multitenancy/FSHTenantInfo.cs

```diff
@@ -40,6 +40,8 @@ public FSHTenantInfo(string id, string name, string? connectionString, string ad
     public bool IsActive { get; private set; }
     public DateTime ValidUpto { get; private set; }
 
+    public bool IsRoot => Id == MultitenancyConstants.Root.Id;
```

Comment:
Can you actually use this property everywhere where we are checking checking for `Id == MultitenancyConstants.Root.Id` now? 

So we can see the usage of this... 

Thinking the name `IsRootTenant` might be more clear... but have to see the usage to know for sure ;-)

Also can you check if there's no migration generated for this... otherwise we will probably have to add the `[NotMapped]` attribute (or specify that in fluent configuration).